test/dotnet-new.Tests/GivenThatIWantANewFSApp.cs

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.Build.Construction;
+using Microsoft.DotNet.Tools.Test.Utilities;
+using Xunit;
+using FluentAssertions;
+
+namespace Microsoft.DotNet.New.Tests
+{
+    public class GivenThatIWantANewFSApp : TestBase
+    {
+        [Fact(Skip="https://github.com/dotnet/cli/issues/4381")]
+        public void When_NewtonsoftJson_dependency_added_Then_project_restores_and_runs()
+        {
+            var rootPath = TestAssetsManager.CreateTestDirectory().Path;
+            var projectName = new DirectoryInfo(rootPath).Name;
+            var projectFile = Path.Combine(rootPath, $"{projectName}.csproj");
+
+            new TestCommand("dotnet") { WorkingDirectory = rootPath }
+                .Execute("new --lang f#");
+            
+            GivenThatIWantANewCSApp.AddProjectDependency(projectFile, "Newtonsoft.Json", "7.0.1");
```

Comment:
This needs to be extracted to another class. The cross-dependency F# on C# is not right...